src/libraries/System.IO.Hashing/src/System/IO/Hashing/Crc32.Vectorized.cs

```diff
@@ -17,7 +18,9 @@ public partial class Crc32
         private static bool CanBeVectorized(ReadOnlySpan<byte> source) =>
             BitConverter.IsLittleEndian
             && VectorHelper.IsSupported
-            && source.Length >= Vector128<byte>.Count * 4;
+            // Vectorization can process spans as short as a single vector (16 bytes), but if ARM intrinsics are supported they
+            // seem to be more performant for spans less than 8 vectors (128 bytes).
+            && source.Length >= Vector128<byte>.Count * (System.Runtime.Intrinsics.Arm.Crc32.IsSupported ? 8 : 1);
```

Comment:
Is JIT smart enough to turn `Vector128<byte>.Count * (System.Runtime.Intrinsics.Arm.Crc32.IsSupported ? 8 : 1)` into a constant?

cc @EgorBo

Reply:
> Is JIT smart enough to turn `Vector128<byte>.Count * (System.Runtime.Intrinsics.Arm.Crc32.IsSupported ? 8 : 1)` into a constant?
> 
> cc @EgorBo

Just checked: folded to a constant

Reply:
@EgorBo thank you very much!

Reply:
It was also one of my concerns. My testing in sharplab.io also shows it folding into a constant.